third_party/2and3/yaml/loader.pyi

```diff
@@ -8,6 +8,9 @@ from yaml.resolver import BaseResolver, Resolver
 class BaseLoader(Reader, Scanner, Parser, Composer, BaseConstructor, BaseResolver):
     def __init__(self, stream) -> None: ...
 
+class FullLoader(Reader, Scanner, Parser, Composer, SafeConstructor, Resolver):
```

Comment:
This actually derives from `FullConstructor`, which is a sub-class of `SafeConstructor`. Since we don't have stubs for that class, yet, I see two options:
* Add `FullConstructor` to `yaml.constructor.pyi`, possible marked with `# incomplete` and use that.
* Just add a comment explaining that `FullConstructor` would be correct here.

Reply:
Doh!  Thanks for catching that.  Fixed in a second commit.